repo/packages/K/kafka/1/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "kafka",
+  "version": "0.9.2.0",
+  "scm": "https://github.com/mesos/kafka",
+  "maintainer": "support@mesosphere.io",
+  "description": "Apache Kafka running on top of Apache Mesos",
+  "framework": true,
+  "tags": ["mesosphere", "framework", "bigdata"],
```

Comment:
we need to remove mesosphere and framework from the tags.  this is used for user search and is intended to differentiate the services.  I would suggest:  "analytics", "queue", "pipeline" anything you imagine a user would search for that you would like kafka to be an answer for... or involved with.